isisd/isis_tlvs.c

```diff
@@ -3228,12 +3275,13 @@ static int pack_items_(uint16_t mtid, enum isis_tlv_context context,
 
 	last_len = len = 0;
 	for (item = item ? item : items->head; item; item = item->next) {
-		rv = pack_item(context, type, item, s, fragment_tlvs, pe, mtid);
-		if (rv)
+		rv = pack_item(context, type, item, s, &min_len, fragment_tlvs,
+			       pe, mtid);
+		if (rv) {
 			goto too_long;
+		}
 
 		len = stream_get_endp(s) - len_pos - 1;
-
```

Comment:
nit: could we leave the blank separator line in place?

Reply:
fixed